Gruntfile.js

```diff
@@ -375,7 +386,8 @@ module.exports = function (grunt) {
             '.htaccess',
             '*.html',
             'images/{,*/}*.{webp}',
-            'styles/fonts/{,*/}*.*'
+            'styles/fonts/{,*/}*.*',
+            'common/application/resources/**/*.json'
```

Comment:
Adding this line means that we don't have to manually copy over the `app.json` file as described in the README [here](https://github.com/esaude/esaude-emr-poc/blame/master/README.md#L130). Is there any problem with doing it this way @valvijo?